### PR TITLE
fix - login security issue 

### PR DESCRIPTION
**Pull request by thareekanvar**
Check the post #1174

### CODE REVIEW

**Comment by boojack** [comment]
@thareekanvar Please fix the eslint error

**Comment by thareekanvar** [comment]
@boojack i think i fixed the TS Lint issue

File: server/auth.go [hunk 1]
```diff
@@ -31,18 +31,18 @@ func (s *Server) registerAuthRoutes(g *echo.Group) {
 		}
 		user, err := s.Store.FindUser(ctx, userFind)
 		if err != nil && common.ErrorCode(err) != common.NotFound {
-			return echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to find user by username %s", signin.Username)).SetInternal(err)
+			return echo.NewHTTPError(http.StatusInternalServerError, "Your login credentials are incorrect please check")
```
Comment (boojack) [inline server/auth.go:34]: How about `Incorrect login credentials, please try again`

Comment (thareekanvar) [reply server/auth.go]: @boojack changed to  `Incorrect login credentials, please try again`  u can check

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**